dbt_style_guide.md

```diff
@@ -1,237 +1,415 @@
+  - Models in the **staging** folder should use the source's name as the `<source/topic>` and the entity name as the `additional_context`.
+
+    Examples:
+    - seed_snowflake_spend.csv
+    - base_stripe__invoices.sql
+    - stg_stripe__customers.sql
+    - int_payments.sql
+    - int_customers__unioned.sql
+    - fct_orders.sql
+
+## Model Organization  
+Our models (typically) fit into three main categories: staging, marts, and base/intermediate.  
+
+For more detail about why we use this structure, check out [this discourse post](https://discourse.getdbt.com/t/how-we-structure-our-dbt-projects/355). The file and naming structures are as follows:
 ```
 ├── dbt_project.yml
 └── models
     ├── marts
-    |   └── core
-    |       ├── intermediate
-    |       |   ├── intermediate.yml
-    |       |   ├── customers__unioned.sql
-    |       |   ├── customers__grouped.sql
-    |       └── core.yml
-    |       └── core.docs
-    |       └── dim_customers.sql
-    |       └── fct_orders.sql
+    |   ├── core
+    |   |   ├── _docs_core.md
+    |   |   ├── _models_core.yml
+    |   |   ├── dim_customers.sql
+    |   |   └── fct_orders.sql
+    |   └── intermediate
+    |       ├── _models_int_core.yml
+    |       ├── int_payments.sql
+    |       ├── int_customers__unioned.sql
+    |       └── int_customers__grouped.sql
     └── staging
         └── stripe
             ├── base
-            |   ├── base__stripe_invoices.sql
-            ├── src_stripe.yml
-            ├── src_stripe.docs
-            ├── stg_stripe.yml
+            |   ├── _models_base_stripe.yml
+            |   └── base_stripe__invoices.sql
+            ├── _docs_stripe.md
+            ├── _models_stripe.yml
+            ├── _sources_stripe.yml
             ├── stg_stripe__customers.sql
             └── stg_stripe__invoices.sql
 ```
-- All objects should be plural, such as: `stg_stripe__invoices`
-- Base tables are prefixed with `base__`, such as: `base__<source>_<object>`
-- Intermediate tables should end with a past tense verb indicating the action performed on the object, such as: `customers__unioned`
-- Marts are categorized between fact (immutable, verbs) and dimensions (mutable, nouns) with a prefix that indicates either, such as: `fct_orders` or `dim_customers`
-
 ## Model configuration
 
-- Model-specific attributes (like sort/dist keys) should be specified in the model.
-- If a particular configuration applies to all models in a directory, it should be specified in the `dbt_project.yml` file.
-- In-model configurations should be specified like this:
-
-```python
-{{
-  config(
-    materialized = 'table',
-    sort = 'id',
-    dist = 'id'
-  )
-}}
-```
-- Marts should always be configured as tables
+- Model configurations at the [group level](https://docs.getdbt.com/reference/model-configs#configuring-directories-of-models-in-dbt_projectyml) should be considered (and if applicable, applied) first.

```

Comment:
Maybe "folder level" instead of "group level"

Reply:
[Done](https://github.com/dbt-labs/corp/pull/63/commits/94a41d4acbebbb12a8c7455b87a5abf3624aaa4f)